llama_index/service_context.py

```diff
@@ -205,7 +205,7 @@ def from_defaults(
             or node_parser
             or _get_default_node_parser(
                 chunk_size=chunk_size or DEFAULT_CHUNK_SIZE,
-                chunk_overlap=chunk_overlap or SENTENCE_CHUNK_OVERLAP,
+                chunk_overlap=SENTENCE_CHUNK_OVERLAP if chunk_overlap is None else chunk_overlap,
```

Comment:
@logan-markewich since we are deprecating service context. better to let you handle it

Reply:
@Miroun are those `chunk_overlap=chunk_overlap or SENTENCE_CHUNK_OVERLAP` the same as `chunk_overlap=SENTENCE_CHUNK_OVERLAP if chunk_overlap is None else chunk_overlap,
`